servant-examples/auth-combinator/auth-combinator.hs

```diff
@@ -27,9 +27,10 @@ data AuthProtected
 
 instance HasServer rest => HasServer (AuthProtected :> rest) where
   type ServerT (AuthProtected :> rest) m = ServerT rest m
+  type HasCfg (AuthProtected :> rest) c = HasCfg rest c
 
-  route Proxy subserver = WithRequest $ \ request ->
-    route (Proxy :: Proxy rest) $ addAcceptCheck subserver $ cookieCheck request
+  route Proxy p subserver = WithRequest $ \ request ->
+    route (Proxy :: Proxy rest) p $ addAcceptCheck subserver $ cookieCheck request
```

Comment:
This example file wasn't using the new Config mechanism at all. Now it is: 9079c55. (And I called it `config`.)